src/modules/blockchainApplication/explore/components/BlockchainApplications/BlockchainApplications.js

```diff
@@ -1,57 +1,19 @@
-import React, { useState } from 'react';
+import React from 'react';
 import { useTranslation } from 'react-i18next';
-import { Swiper, SwiperSlide } from 'swiper/react';
-import { Pagination, Navigation } from 'swiper';
 import TabsContainer from '@theme/tabs/tabsContainer/tabsContainer';
-import InfoBanner from '@common/components/infoBanner/infoBanner';
 import BlockchainApplicationList from '@blockchainApplication/explore/components/BlockchainApplicationList';
+import SwippableInfoBanner from '@common/components/infoBanner/swippableInfoBanner';
 import BlockchainApplicationStatistics from '../BlockchainApplicationStatistics';
 import styles from './BlockchainApplications.css';
 import banners from './banners';
 import SessionManager from '../../../connection/components/SessionManager';
 
 const BlockchainApplications = ({ applications, statistics, applyFilters, filters }) => {
   const { t } = useTranslation();
-  const [sliderVisibility, setSliderVisibility] = useState(
-    !localStorage.getItem('blockchainApplicationsPageBanner')
-  );
-  const handleSliderBannerClose = () => setSliderVisibility(!sliderVisibility);
 
   return (
     <div className={styles.wrapper}>
-      {sliderVisibility && (
-        <Swiper
-          pagination={{ clickable: true }}
-          navigation
-          modules={[Pagination, Navigation]}
-          loop
-          slidesPerView="auto"
-          spaceBetween={20}
-          className={styles.bannerSwiper}
-        >
-          {banners.map((bannerInfo, index) => {
-            const { infoMessage, infoDescription, illustrationName, infoLink, infoLinkText } =
-              bannerInfo;
-            return (
-              <SwiperSlide key={index}>
-                <InfoBanner
-                  t={t}
-                  className={styles.bannerWrapper}
-                  name="blockchainApplicationsPageBanner"
-                  infoLabel={t('New')}
-                  infoMessage={infoMessage(t)}
-                  infoDescription={infoDescription(t)}
-                  illustrationName={illustrationName}
-                  handleSliderBannerClose={handleSliderBannerClose}
-                  infoLink={infoLink}
-                  infoLinkText={infoLinkText}
-                  show
-                />
-              </SwiperSlide>
-            );
-          })}
-        </Swiper>
-      )}
+      <SwippableInfoBanner banners={banners} />
```

Comment:
NRTTL: Please remove `swiperBanner` styles in the CSS file as it's no longer used directly in this component and is a duplicate of the similar style in SwippableBanner component.

Reply:
Resolved